test/redeyed-smoke.js

```diff
@@ -21,7 +21,7 @@ test('tap', function (t) {
     , 'resolve/test/precedence/'
     , 'is-relative/index.js'
     , 'is-buffer/test/'
-    , 'lodash/utility/'
+    , 'lodash/'
```

Comment:
Expand the exclusion of any lodash files.

This is the fragility of the smoke tests approach. A file `lodash/constant.js`:
```javascript
/**
 * Creates a function that returns `value`.
 *
 * @static
 * @memberOf _
 * @since 2.4.0
 * @category Util
 * @param {*} value The value to return from the new function.
 * @returns {Function} Returns the new constant function.
 * @example
 *
 * var objects = _.times(2, _.constant({ 'a': 1 }));
 *
 * console.log(objects);
 * // => [{ 'a': 1 }, { 'a': 1 }]
 *
 * console.log(objects[0] === objects[1]);
 * // => true
 */
function constant(value) {
  return function() {
    return value;
  };
}
```
will trigger a false alarm since it contains the string `"var"` in the source input and yet that is part of the comment hence it won't be _surrounded_ by redeyed.

This has never been a problem before. However, this version of lodash is transitively included due to some other updates on a dependency (a by-product of loose dependency tracking via `~` or `^`).